docs/_docs/collections.md

```diff
@@ -198,7 +198,7 @@ each of the following `permalink` configurations will produce the document struc
         <p><code>:title</code></p>
       </td>
       <td>
-        <p>The document's lowercase title (as defined in its <a href="/docs/frontmatter/">front matter</a>), with every sequence of spaces and non-alphanumeric characters replaced by a hyphen. If the document does not define a title in its <a href="/docs/frontmatter/">front matter</a>, this is equivalent to <code>name</code>.</p>
+        <p>The document's lowercase title (as defined in its <a href="/docs/frontmatter/">front matter</a>), with every sequence of spaces and non-alphanumeric characters replaced by a hyphen. If the document inside of a collection does not define a <code>slug</code> in its <a href="/docs/frontmatter/">front matter</a>, this is equivalent to <code>name</code>.</p>
```

Comment:
`inside of a collection` is redundant: this text is on the `collections` documentation page so it's implied. 😃 

From [the code](https://github.com/jekyll/jekyll/blob/73419cb374be1b8f45818a23116cf71db93549ce/lib/jekyll/drops/url_drop.rb#L22-L25), it looks like it never uses the `title` front matter value. Perhaps this paragraph needs to be rewritten:

> The document's `slug` (as defined in its front matter). If the document does not define `slug` in its front matter, this is equivalent to `:name`.